draft-ietf-rats-corim.md

```diff
@@ -1368,12 +1368,16 @@ The Conditional Endorsement Series Triple has the following structure:
 The `conditional-endorsement-series-triple-record` has the following parameters:
 
 * `condition`: Search criteria that locates Evidence, corroborated Evidence, or Endorsements.
+The condition is a record containing an `environment`, optional `claims-list`, and optional `authorized-by`.
```

Comment:
@nedmsmith This text is not aligned with what CDDL renders itself to.
In CDDL each claim is having an Authority which seems incorrect.

To me each of the condition: for each of the `environment`should have the authority

Reply:
The text here at line 1371 reads:
`The condition is a record containing an `environment`, optional `claims-list`, and optional `authorized-by`.` 

While CDDL says

Reply:
@nedmsmith please see my comments above